packages/tree-widget/README.md

```diff
@@ -180,3 +180,20 @@ export class TreeWidgetControl extends WidgetControl {
   }
 }
 ```
+
+### Using the TreeWidgetUiItemsProvider
+
+The `TreeWidgetUiItemsProvider` functions like the `TreeWidgetControl` but is instead passed through the `uiProviders` prop of the `<Viewer>` component.
+
+```ts
+<Viewer
+  authConfig={{ oidcClient: AuthorizationClient.oidcClient }}
```

Comment:
i'd honestly remove all the other unused props here. take a look at the snippet i added here https://github.com/imodeljs/viewer-components-react/tree/master/packages/measure-tools#what-to-add-in-your-itwin-viewer-based-app-itwin-app-ui